docs/platform/tutorials/image-processing/add-service.md

```diff
@@ -0,0 +1,235 @@
+# 👩‍🔬 Lab - Add a new service
+
+In this lab you use everything you've learned so far, to add a new service to the pipeline. Specifically, you add a service to publish the number of cars captured by the TfL cams to a new topic. You will then observe the number of cars change in real time using the waveform view of the Quix Data Explorer. This service could be useful if you want to easily store the number of cars, or perhsp create an alarm if the number of cars rises above a certain threshold. This service is a simple example of filtering - where you filter out data you are not interested in for subsequent processing.
+
+You develop this service on a feature branch, and then, you create a PR to merge your new feature into the develop branch. This is a common pattern for development - you can test your new service on the feature branch, and then test again on the develop branch, before final integration into the production `main` branch.
+
+## Create an environment
+
+To create a new environment (and branch):
+
+1. Click `+ New environment` to create a new environment (note, your screen will look slightly different to the one shown here):
+
+    ![New environment](../image-processing/images/new-environment.png)
+
+2. Create a new environment called `Cars Only`.
+
+3. Create a new branch called `cars-only`. To do this, from the branch dropdown click `+ New branch` which displays the New branch dialog:
+
+    ![New branch](../image-processing/images/new-branch.png)
+
+    !!! important
+
+        Make sure you branch from the `develop` branch, not `main`, as you are going to merge your changes onto the `develop` branch.
+
+4. Complete creation of the environment using the default options.
+
+5. On the projects screen, click your newly created environment, `Cars Only`.
+
+## Sync the environment
+
+You now see that the Quix environment is out of sync with the Git repository. You need to synchronize the Quix view of the environment, with that stored in the repository. 
+
+To synchronize Quix with the repository:
+
+1. Click `Sync environment`:
+
+    ![Sync environment](../image-processing/images/sync-environment.png)
+
+    The sync environment dialog is displayed, showing you the changes that are to be made to the `quix.yaml` file, which is the configuration file that defines the pipeline.
+
+2. Click `Sync environment`, and then `Go to pipeline`. 
+
+    In the pipeline view, you see the services building. Ensure all services are "Running" before continuing.
+
+## Add a transform
+
+You now add a transform to the output of the stream merge service. This is a convenient point, as the multiple streams are now merged to one stream (all cameras are merged into one stream), and this will make viewing the number of cars easier in the waveform view of the data explorer, as there is only one stream to examine.
```

Comment:
I found this part a big confusing, since at the beginning, the docs say "this is legacy and we shall not utter it's name again"... but here we are using it for an exercise. Does that mean it's staying in, and serves some kind of functional role?

Reply:
Stream merge service is staying (latest). I've added new text to describe it in the overview, added it to the list of services, and added a new section in "other services".